magenta/models/arbitrary_image_stylization/arbitrary_image_stylization_build_model.py

```diff
@@ -130,6 +131,8 @@ def style_prediction(style_input_,
     style_prediction_bottleneck: int. Specifies the bottleneck size in the
         number of parameters of the style embedding.
     reuse: bool. Whether to reuse model parameters. Defaults to False.
+    inception_only: bool. If true, does not build the mapping from bottleneck
```

Comment:
Is there a reason why one would need to avoid constructing that part of the computation graph rather than ignoring the `style_params` dict returned by this function?

Reply:
No reason other than I didn't think of it :)